ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-dev-preview.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.13"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.13"
+  tools:
+    name: "4.13"
+    namespace: ocp
+    tag: tools
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
```

Comment:
We do not use the version for provisioning an OSD cluster, we get the version in our steps. Is the 'releases' part required for the yaml format?

If the 'releases' part is required, due to now the DPTP does not support get version from the dev-preview and the EC build is a kind of nightly build,  the item 'stream' is set to 'nightly' to not block the OSD jobs. When the OCP side is ready, it can be changed to fit its requirement.